### PR TITLE
Emotion 'styled-components' mentions cleanup

### PR DESCRIPTION
**Pull request by RichardPK**
Resolves #NUMBER

**Overall change:** _A very high-level summary of easily-reproducible changes that can be understood by non-devs._

**Code changes:**

- _A bullet point list of key code changes that have been made._
- _When describing code changes, try to communicate **how** and **why** you implemented something a specific way, not just **what** has changed._

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-brand/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 7.0.2 | [PR#?](https://github.com/bbc/psammead/pull/?) Remove unnecessary comment and linting |
```
Comment (RichardPK) [inline packages/components/psammead-brand/CHANGELOG.md]:
```suggestion
| 7.0.2 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove unnecessary comment and linting |
```

File: packages/components/psammead-media-player/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version       | Description                                                                                                                  |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| 5.0.1 | [PR#?](https://github.com/bbc/psammead/pull/?) Remove styled components reference in comments |
```
Comment (RichardPK) [inline packages/components/psammead-media-player/CHANGELOG.md]:
```suggestion
| 5.0.1 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove styled components reference in comments |
```

File: packages/components/psammead-navigation/CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 8.0.1| [PR#?](https://github.com/bbc/psammead/pull/?) Remove comment referencing styled-components. |
```
Comment (RichardPK) [inline packages/components/psammead-navigation/CHANGELOG.md]:
```suggestion
| 8.0.1| [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove comment referencing styled-components. |
```

File: packages/components/psammead-script-link/CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.0.1 | [PR#?](https://github.com/bbc/psammead/pull/?) Remove comment referencing styled-components |
```
Comment (RichardPK) [inline packages/components/psammead-script-link/CHANGELOG.md]:
```suggestion
| 3.0.1 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove comment referencing styled-components |
```

File: packages/components/psammead-useful-links/CHANGELOG.md [hunk 5]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.0.1 | [PR#?](https://github.com/bbc/psammead/pull/?) Remove styled components reference |
```
Comment (RichardPK) [inline packages/components/psammead-useful-links/CHANGELOG.md]:
```suggestion
| 3.0.1 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove styled components reference |
```

File: packages/utilities/psammead-test-helpers/CHANGELOG.md [hunk 6]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 5.0.1 | [PR#?](https://github.com/bbc/psammead/pull/?) Remove reference to styled-components in pacakge description |
```
Comment (RichardPK) [inline packages/utilities/psammead-test-helpers/CHANGELOG.md]:
```suggestion
| 5.0.1 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove reference to styled-components in pacakge description |
```

File: packages/components/psammead-navigation/CHANGELOG.md [hunk 7]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 8.0.1| [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove comment referencing styled-components. |
```
Comment (simonsinclair) [inline packages/components/psammead-navigation/CHANGELOG.md]:
```suggestion
| 8.0.1 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove comment referencing styled-components. |
```

**Review by ryanmccombe — APPROVED** [review]

**Review by simonsinclair — APPROVED** [review]
All good - maybe we should make the style-lint change more apparent in the changelog? 👍

File: packages/components/psammead-brand/CHANGELOG.md [hunk 8]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
+| 7.0.3 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove unnecessary comment and linting |
```
Comment (RichardPK) [inline packages/components/psammead-brand/CHANGELOG.md]:
```suggestion
| 7.0.3 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove styled-component 'referencing other components' comment and reenabled stylelint for the style block that references another component  |
```

File: packages/components/psammead-media-player/CHANGELOG.md [hunk 9]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version       | Description                                                                                                                  |
 | ------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| 5.0.2 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove styled components reference in comments |
```
Comment (RichardPK) [inline packages/components/psammead-media-player/CHANGELOG.md]:
```suggestion
| 5.0.2 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove styled-component 'referencing other components' comment and reenabled stylelint for the style block that references another component |
```

File: packages/components/psammead-script-link/CHANGELOG.md [hunk 10]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 3.0.2 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove comment referencing styled-components |
```
Comment (RichardPK) [inline packages/components/psammead-script-link/CHANGELOG.md]:
```suggestion
| 3.0.2 | [PR#3927](https://github.com/bbc/psammead/pull/3927) Remove styled-component 'referencing other components' comment and reenabled stylelint for the style block that references another component |
```

**Pull request merged**